fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/JcrPropertyStatementListener.java

```diff
@@ -193,19 +195,10 @@ private void validateSubject(final Resource subject) {
 
     /**
      * Assert that no exceptions were thrown while this listener was processing change
-     * @throws MalformedRdfException if malformed rdf exception occurred
-     * @throws javax.jcr.AccessDeniedException if access denied exception occurred
      */
-    public void assertNoExceptions() throws MalformedRdfException, AccessDeniedException {
+    public void assertNoExceptions() {
         if (!exceptions.isEmpty()) {
-            final StringJoiner sb = new StringJoiner("\n");
-            for (final Exception e : exceptions) {
-                sb.add(e.getMessage());
-                if (e instanceof AccessDeniedException) {
-                    throw new AccessDeniedException(sb.toString());
```

Comment:
I am concerned that we have lost the `AccessDeniedException` scenario resolved with this issue:
https://jira.duraspace.org/browse/FCREPO-1379